clients/user-services/jdbc-user-service/src/main/java/com/github/ymikevich/service/utils/DatabaseRequestFunctionExecutor.java

```diff
@@ -0,0 +1,60 @@
+package com.github.ymikevich.service.utils;
+
+import com.github.ymikevich.exceptions.DatabaseOperationException;
+import com.github.ymikevich.service.connection.pool.HikariCPDatasource;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public interface DatabaseRequestFunctionExecutor {
```

Comment:
methods in this interface execute not only "function", but also the "consumer" requests. I'd like to think about naming